README.md

```diff
@@ -1,21 +1,94 @@
+
+Developers rarely work all by themselves, on a deadline, or ship something they’ll only use once (with no idea whether it actually works). 
+ 
+Wouldn’t students be better served by showing versions of their work, iterating, checking in on milestones and showing off the final product? 
+ 
+With GitHub Classroom you can set up the industry-standard workflow and free up your time to focus on teaching. 
+ 
+*Classroom is a teacher-facing tool that uses the GitHub API to enable the GitHub workflow for education.* 
+ 
+You create an Assignment with starter code and directions, send along one link, and students get their own “sandbox” copy of the repo to get started.
+ 
+Set due dates, track assignments in your teacher dashboard, or integrate other tools like testing frameworks. 
+ 
+With Classroom, you can spin up your course on GitHub and move on to the good stuff. 
+ 
+@johndbritton, @mozzadrella, @nwoodthorpe, @tarebyte, and @wilfriedE are all maintainers. 
 
-### Help wanted
+![GitHub Classroom screenshot](https://cloud.githubusercontent.com/assets/1311594/14748352/32f677b0-0887-11e6-9ac2-8aa06e4341fa.png)
 
-If you're interested in helping out with Classroom development and looking for a place to get started, check out the issues labeled [`help-wanted`](https://github.com/education/classroom/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp-wanted) and feel free to ask any questions you have before diving into the code.
+## Why try Classroom?
+ 
+*Spend more time with students, less on setup.* Students accept an assignment with one link, so you can get straight to the material. 
+ 
+*Bootstrap group assignments in a snap.* Invite students to a shared repository, and cap the number of students per group. Use the same groups over and over again, or create new ones.
+ 
+*More insight into student work than ever before.* See when students accept the assignment, and access their work from the moment they start. With version control, catch when they get stuck and help them rewind. 
+ 
+*You are in control.* Students can work individually or in groups, in public or in private. Set permissions for teaching assistants or graders. 
+ 
+*Scales for large courses with ease.* If you have a small course, Classroom will make your life easier and save you time. f you have hundreds of students, we have your covered: as many repositories as you need, and webhooks to integrate automated testing tools. 
+ 
+*Works with your Learning Management System (LMS).* Students submit a link to their assignment repository to your learning management system. Give feedback through comments in GitHub, but keep grades in your LMS.
+ 
+*You’re not alone.* The experts on the GitHub Education team are here to answer any of your questions, and we’ve got docs, best practices, and a strong community of educators to help you migrate to Classroom.
+ 
+*Are you super-advanced?* Do you want to build your own tools? We 💖 contributions. 
```

Comment:
What do you think about having a contribution guide that we link here, or maybe the "how to setup classroom" doc? Right now this seems a little open ended